composer.json

```diff
@@ -131,5 +133,13 @@
 		"wpackagist-theme/twentysixteen": "2.0",
 		"wpackagist-theme/twentyseventeen": "2.2",
 		"wpackagist-theme/twentynineteen": "1.4"
+	},
+	"scripts": {
+		"phpcs": [
+			"phpcs --extensions=php -s -p"
```

Comment:
`MAYBE`: `-a` would be useful when running manually IMO

Reply:
What does `-a` do?

Reply:
It's the interactive mode, so after you fix a bunch you can hit enter and it re-checks everything. If there are more left it the current file then it shows you the updated list, and if there aren't then it moves on the next file.

Reply:
Since this is the command that's also used for travis, I don't think we should add it. You can pass extra flags through to phpcs with the `--` separator, though, so this will work for you:

`composer run-script phpcs [path] -- -a`